### ISSUE TITLE
Page references not updated if user does not hit enter

### ISSUE THREAD

**Issue by dudelson** [athensresearch/athens#1423]
If I have a page with references elsewhere, and I change the title of the page, and after I'm done changing the page title I click into another block instead of hitting enter, the page references are not updated.

Easy workaround is of course to remember to hit enter to finalize the title change, but I have a habit of immediately moving on to the next thing by clicking elsewhere. I'd like the page title change to finalize when the title loses focus so it works in every case.

**Comment by juniusfree** [comment]
Hi @dudelson I can't replicate the issue. Can you verify if this is how it's supposed to be replicated?

![issues-1423](https://user-images.githubusercontent.com/8164667/126915874-c8cdb7a7-bc97-4f2b-bffd-bdb5ad628e04.gif)

**Comment by dudelson** [comment]
Yeah that's exactly it. Please let me know what other info I can provide to help troubleshoot.

**Comment by dudelson** [comment]
@juniusfree I was able to replicate the results of your test above. The bug does not appear when I create a new page for test purposes. But I also just tried it on a page I've had for a while (and actually use, thus several block refs distributed across time), and the bug appeared in that case. So perhaps it has something to do with the age of a page, or whether the page was created prior to a certain release of athens, or something like that.
